### PR TITLE
Update JSDOM to 10.1.x

### PR DESCRIPTION
**Pull request by ellatrix**

### CODE REVIEW

File: bootstrap-test.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
+/* eslint-disable no-var */
```
Comment (aduth) [inline bootstrap-test.js]: We can and should use `const` here.

http://node.green/#ES2015-bindings-const

File: bootstrap-test.js [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
+
 // Chai plugins
 require( 'chai' )
 	.use( require( 'dirty-chai' ) )
 	.use( require( 'sinon-chai' ) );
 
 // Fake DOM
-global.document = require( 'jsdom' ).jsdom( '', {
+var JSDOM = require( 'jsdom' ).JSDOM;
```
Comment (aduth) [inline bootstrap-test.js]: Minor: Could assign by destructure:

```js
const { JSDOM } = require( 'jsdom' );
```

**Review by aduth — CHANGES_REQUESTED** [review]

File: bootstrap-test.js (hunk 1 above)
Comment (iseulde) [inline bootstrap-test.js]: Huh, strange, I tried this before and it errored.

File: bootstrap-test.js [hunk 3]
```diff
@@ -4,16 +4,19 @@ require( 'chai' )
 	.use( require( 'sinon-chai' ) );
 
 // Fake DOM
-global.document = require( 'jsdom' ).jsdom( '', {
+const { JSDOM } = require(	 'jsdom' );
```
Comment (aduth) [inline bootstrap-test.js]: Tab snuck in instead of space after `require(`

Comment (iseulde) [inline bootstrap-test.js]: Whoops.

Comment (iseulde) [inline bootstrap-test.js]: I'm surprised eslint didn't catch this.

**Review by aduth — APPROVED** [review]

**Pull request merged**